rudder-inventory-ldap/SPECS/rudder-inventory-ldap.spec

```diff
@@ -290,9 +290,19 @@ fi
 
 # Need to restart to take schema changes into account
 echo -n "INFO: Restarting rudder-slapd..."
-/sbin/service rudder-slapd restart >/dev/null 2>&1
+/sbin/service rudder-slapd force-reload >/dev/null 2>&1
```

Comment:
force-reload basically does the same thing as restart, but instead of stop,start, it's force-stop,start:

```
  restart)
  stop_slapd
  stop_slurpd
  configtest
  start_slurpd
  start_slapd
  ;;
  force-reload)
  forcestop
  configtest
  start_slurpd
  start_slapd
  ;;
\\\